components/fibery/actions/list-entities/list-entities.mjs

```diff
@@ -0,0 +1,15 @@
+import common from "../common/common-entities.mjs";
+
+delete common.props.attributes;
+
+export default {
+  ...common,
+  key: "fibery-list-entities",
+  name: "List Entities",
+  description: "Lists entities for a type. [See the docs here](https://api.fibery.io/graphql.html#list-of-entities)",
+  version: "0.0.1",
+  type: "action",
+  async run({ $ }) {
+    return this.findEntities($);
```

Comment:
I think this could be adjusted to export a summary in the same way `list-types` was done